.gitmodules

```diff
@@ -5,10 +5,9 @@
 # TODO [LSM]: Revert to cosmos relayer once they've merged 
 # the fix for the unbonding time query
 [submodule "deps/relayer"]
-	# Branch: sam/lsm-poc
-	# Commit: 0136134766142f133f263f3bba72d7077e7bc2c1
+	# Commit: v2.4.2
 	path = deps/relayer
-	url = https://github.com/Stride-Labs/relayer.git
+	url = https://github.com/cosmos/relayer.git
```

Comment:
Are these changes to .gitmodules intentional?

Reply:
yeah, but I can move it to a separate PR if you'd prefer. They were small so I thought I'd just add them to this one

Reply:
no its fine just checking that they're intentional